### ISSUE TITLE
UPE Appearance on the Shop theme results in mismatched background color

### ISSUE THREAD

**Issue by pierorocca** [Automattic/woocommerce-payments#9796]
### Describe the bug
The wrong background color is computed causing the payments section to stand out and for the grey borders to get lost in the grey background.

![Image](https://github.com/user-attachments/assets/ac2ef405-ba8e-470e-9cfd-32f3fff60202)

1. Install the Shoper theme by ThemeArt
2. Shop
3. Observe checkout background is white, payments components background is grey/

**Comment by gpressutto5** [comment]
Hi Piero, after some investigation I confirmed the Payment Methods list background is supposed to be grey, and this is unrelated to UPE styling.

![Image](https://github.com/user-attachments/assets/1802c3b6-4bb8-413a-b218-75bb8df45c90)

-----

The other issue I noticed is that the UPE inputs background looks different from the inputs from the checkout form. I confirmed this is also not an issue with how we compute the styling. The theme uses RGBA colors for the inputs background so they look different based on the background of the div they're in.

![Image](https://github.com/user-attachments/assets/9a59cb16-13a0-4776-8402-62ad2759cfa4)
